dvclive/live.py

```diff
@@ -130,8 +133,7 @@ def set_step(self, step: int) -> None:
 
         self.make_report()
 
-        if self._checkpoint:
-            make_checkpoint()
+        self.make_checkpoint()
```

Comment:
Always call it, the `env.DVC_CHECKPOINT` is what would determine if we actually generate checkpoint or not. This allows to work with checkpoints even without the `live` section